pages/about.vue

```diff
@@ -3,56 +3,44 @@
 		<h2 class="title"><span>About</span></h2>
 		<h3>Period</h3>
 		<p class="period">
-			<a href="https://www.timeanddate.com/worldclock/fixedtime.html?iso=20231104T16&p1=248" target="_blank" rel="noopener noreferrer">11/04 07:00</a>
+			<a href="https://www.timeanddate.com/worldclock/fixedtime.html?iso=20231104T16&p1=248" target="_blank" rel="noopener noreferrer"> 08/11 18:00 </a>
 			-
-			<a href="https://www.timeanddate.com/worldclock/fixedtime.html?iso=20231105T16&p1=248" target="_blank" rel="noopener noreferrer">11/05 07:00</a>
+			<a href="https://www.timeanddate.com/worldclock/fixedtime.html?iso=20231105T16&p1=248" target="_blank" rel="noopener noreferrer"> 10/11 18:00 </a>
 		</p>
 		<p :style="{textAlign: 'center'}">(UTC)</p>
 		<h3>Socials</h3>
 		<ul class="socials">
 			<li>
 				<iframe
-					src="https://discord.com/widget?id=546339917459095552&theme=dark"
+					src="https://discord.com/widget?id=1261029732582162442&theme=dark" 
 					allowtransparency="true"
 					frameborder="0"
 					width="100%"
 					height="100%"
 				>
-					<a href="https://discord.gg/xcZ3t9bC2A" target="_blank" rel="noopener noreferrer">Discord</a>
+					<a href="https://discord.gg/2AbXu9sD2G" target="_blank" rel="noopener noreferrer">Discord</a>
 				</iframe>
 			</li>
-			<li>
-				<timeline
-					id="tsgctf"
-					source-type="profile"
-					:options="{tweetLimit: '3', theme: 'dark', height: '500'}"
-				>
-					<a href="https://twitter.com/tsgctf" target="_blank" rel="noopener noreferrer">Tweets from @tsgctf</a>
-				</timeline>
-			</li>
 		</ul>
 		<h3>Sponsor</h3>
-		<a href="https://flatt.tech/" target="_blank" rel="noopener noreferrer">
-			<img src="https://ctf.tsg.ne.jp/images/flatt_image.png" :style="{maxWidth: '60rem'}">
+		<a href="https://www.infomaniak.com/" target="_blank" rel="noopener noreferrer">
+			<img src="./infomaniak.png">
 		</a>
-		<p>Flatt Security Inc. is a cyber security startup based in Tokyo, Japan.</p>
-		<p>We provide a variety of cyber security services, including security assessment.  We also develop and provide security software “Shisho Cloud” for client companies.</p>
-		<p>We are hiring engineers. Please see our employment page <a href="https://recruit.flatt.tech/">https://recruit.flatt.tech/</a> and contact to <a href="https://flatt.tech/contact/">https://flatt.tech/contact/</a></p>
+		<p>Infomaniak is a Swiss-based company specializing in web hosting, cloud services, and domain registration.</p>
+		<p>Known for its focus on privacy, security, and sustainability, it offers solutions for websites, email, and drive.</p>
+		<p>Do not hesitate to take a look to their offers ! <a href="https://www.infomaniak.com/">https://www.infomaniak.com/</a> </p>
 		<h3>Prizes</h3>
-		<ul>
-			<li>1st: 1,000USD</li>
-			<li>2nd: 500 USD</li>
-			<li>3rd: 200 USD</li>
-		</ul>
+
+		<p>TBD</p>
+
```

Comment:
Include the prizes for ik, afawk these are the only prizes we have

Reply:
done